csharp/Platform.Numbers/CalcResult.cs

```diff
@@ -0,0 +1,461 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+namespace Platform.Numbers
+{
+    public class CalcResult<T>
+    {
+        public CalcResult() : this(default) { }
+
+        public CalcResult(T value)
+        {
+            Value = value;
+            IsValid = true;
+        }
+
+        #region Arithmetic operations
+      
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public CalcResult<T> Add(T a)
```

Comment:
What is the reason to create `Add` method at the same time with `+` operator? And if you are so sure it is needed, why you cannot simply call the `+` operator from this method?

Reply:
You're right. It is enough to leave only the + operator with others.